### PR TITLE
Add a link from the BottomAppBar in the session details to the floor map

### LINKED ISSUE #162: Add a link from the BottomAppBar in the session details to the floor map
## Kind (Required)

- Improvement

 ## Overview (Required)

- Just open the FloorMapFragment. 🗺 
image

 ## Links	

-

### PR DESCRIPTION
**Pull request by kkt-yu**
## Issue
- close #162

## Overview (Required)
- Add a link from the BottomAppBar in the session details to the floor map

## Links
- N/A

## Screenshot
Before | After
:--: | :--:
 |

### CODE REVIEW

**Comment by FujiKinaga** [comment]
👀

File: feature/session/src/main/java/io/github/droidkaigi/confsched2020/session/ui/SessionDetailFragment.kt [hunk 1]
```diff
@@ -25,15 +27,11 @@ import io.github.droidkaigi.confsched2020.di.PageScope
 import io.github.droidkaigi.confsched2020.ext.assistedActivityViewModels
 import io.github.droidkaigi.confsched2020.ext.assistedViewModels
 import io.github.droidkaigi.confsched2020.ext.getThemeColor
-import io.github.droidkaigi.confsched2020.model.Session
-import io.github.droidkaigi.confsched2020.model.Speaker
-import io.github.droidkaigi.confsched2020.model.SpeechSession
-import io.github.droidkaigi.confsched2020.model.defaultLang
-import io.github.droidkaigi.confsched2020.model.defaultTimeZoneOffset
```
Comment (takahirom) [inline feature/session/src/main/java/io/github/droidkaigi/confsched2020/session/ui/SessionDetailFragment.kt]: Thank you 🙏
Probably your Project.xml is initialized🧐
We don't use star import by Android Kotlin Style Guide

Comment (kkt-yu) [reply feature/session/src/main/java/io/github/droidkaigi/confsched2020/session/ui/SessionDetailFragment.kt]: Thank you for the review.
Corrected and pushed.

File: feature/session/src/main/java/io/github/droidkaigi/confsched2020/session/ui/SessionDetailFragment.kt [hunk 2]
```diff
@@ -88,6 +90,30 @@ class SessionDetailFragment : DaggerFragment() {
...
+    }
+
+    private fun handleNavigation(@IdRes itemId: Int): Boolean {
+        val navController = findNavController()
+        return try {
+            // ignore if current destination is selected
+            if (navController.currentDestination?.id == itemId) return false
+            val builder = NavOptions.Builder()
+                .setLaunchSingleTop(true)
+                .setPopUpTo(R.id.main, false)
```
Comment (FujiKinaga) [inline feature/session/src/main/java/io/github/droidkaigi/confsched2020/session/ui/SessionDetailFragment.kt]: Thanks for your first-time contribution🎉
It doesn't need an option in SessionDetail to go back to this page after navigating floor map.
Could you please try removing these above two lines and check🙇‍♂️

Comment (kkt-yu) [reply feature/session/src/main/java/io/github/droidkaigi/confsched2020/session/ui/SessionDetailFragment.kt]: Thank you for the review.
Removed and pushed.

**Comment by FujiKinaga** [comment]
@kkt-yu 
Almost LGTM! Thanks for your contribution! :tada:

**Pull request merged**